[CI] SpatialEnvelopeVisitorTests testVisitGeoPointsWrapping failing

**Build Scans:**
- [elasticsearch-intake #22300 / part1](https://gradle-enterprise.elastic.co/s/fdeqgwjcdhtwu)
- [elasticsearch-intake #22286 / part1](https://gradle-enterprise.elastic.co/s/bt75ukkptrvos)

**Reproduction Line:**
```
./gradlew ":libs:geo:test" --tests "org.elasticsearch.geometry.utils.SpatialEnvelopeVisitorTests.testVisitGeoPointsWrapping" -Dtests.seed=8E95F1E415A97FE6 -Dtests.locale=ce-RU -Dtests.timezone=Africa/Timbuktu -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.geometry.utils.SpatialEnvelopeVisitorTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testVisitGeoPointsWrapping),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 82: POINT (-2.9542381370107403E-14 58.7754493653467)
Expected: a value less than or equal to <-2.9542381370107403E-14>
     but: <0.0> was greater than <-2.9542381370107403E-14>
```

**Issue Reasons:**
- [main] 2 failures in test testVisitGeoPointsWrapping (0.8% fail rate in 250 executions)
- [main] 2 failures in step part1 (4.3% fail rate in 46 executions)
- [main] 2 failures in pipeline elasticsearch-intake (4.3% fail rate in 46 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/libs/geo/src/test/java/org/elasticsearch/geometry/utils/SpatialEnvelopeVisitorTests.java
+++ b/libs/geo/src/test/java/org/elasticsearch/geometry/utils/SpatialEnvelopeVisitorTests.java
@@ -134,7 +134,7 @@ public class SpatialEnvelopeVisitorTests extends ESTestCase {
             } else {
                 // Both positive and negative x values exist, we need to decide which way to wrap the bbox
                 double unwrappedWidth = maxPosX - minNegX;
-                double wrappedWidth = (180 - minPosX) - (-180 - maxNegX);
+                double wrappedWidth = 360.0 + maxNegX - minPosX;
                 if (unwrappedWidth <= wrappedWidth) {
                     // The smaller bbox is around the front of the planet, no dateline wrapping required
                     assertRectangleResult(i + ": " + point, result, minNegX, maxPosX, maxY, minY, false);

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -396,9 +396,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.plugin.DataNodeRequestSenderIT
   method: testSearchWhileRelocating
   issue: https://github.com/elastic/elasticsearch/issues/127188
-- class: org.elasticsearch.geometry.utils.SpatialEnvelopeVisitorTests
-  method: testVisitGeoPointsWrapping
-  issue: https://github.com/elastic/elasticsearch/issues/123425
 - class: org.elasticsearch.xpack.remotecluster.CrossClusterEsqlRCS2EnrichUnavailableRemotesIT
   method: testEsqlEnrichWithSkipUnavailable
   issue: https://github.com/elastic/elasticsearch/issues/127368

